src/index.js

```diff
@@ -1,8 +1,53 @@
 import './style.css';
+import { dataPush, dataFetch } from './modules/dataExchange.js';
 
 const refreshBtn = document.querySelector('.refresh');
-const scoreBoard = document.querySelector('.scoreboard');
+const submitBtn = document.querySelector('.submit');
+const nameInput = document.getElementById('name');
+const scoreInput = document.getElementById('score');
+
+const sendData = () => {
+  const scoreData = {
+    user: nameInput.value,
+    score: scoreInput.value,
+  };
+  dataPush(scoreData);
+  nameInput.value = '';
+  scoreInput.value = '';
+};
+
+// event listener for the submit button
+submitBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  sendData();
+});
+
+const renderList = async () => {
+  const data = await dataFetch();
+  const scoreList = document.getElementById('list-body');
+  const listFragment = document.createDocumentFragment();
+  data.result.forEach((data) => {
+    const newList = document.createElement('li');
+    newList.className = 'list';
+
+    const namePara = document.createElement('p');
+    namePara.textContent = `Name: ${data.user}`;
+    const scorePara = document.createElement('p');
+    scorePara.textContent = `Score: ${data.score}`;
+
+    newList.appendChild(namePara);
+    newList.appendChild(scorePara);
+    listFragment.appendChild(newList);
+  });
+  scoreList.appendChild(listFragment);
+};
```

Comment:
[OPTIONAL]

- [ ] I think it will be better if you sort the scores you are receiving in descending order such that the highest score remains at the top while the lowest score is at the bottom just as a proper leaderboard app should be 😄